db/migrate/20190318182503_create_repairs.rb

```diff
@@ -0,0 +1,10 @@
+class CreateRepairs < ActiveRecord::Migration[5.2]
+  def change
+    create_table :repairs do |t|
+      t.string :repair_type
+      t.string :repair_notes
+      t.integer :owner_id
```

Comment:
If `owner_id` is a foreign key, that needs to be indicated in the migration.

Reply:
@Kwasi008 this is correct, we did not cover this very much in class, here is some documentation: https://api.rubyonrails.org/v5.1/classes/ActiveRecord/ConnectionAdapters/TableDefinition.html#method-i-references

You column should be something like

```
t.references :owner, foreign_key: true
```

This explains a little what foreign keys are: https://database.guide/what-is-a-foreign-key/